core/azure-core/src/main/java/com/azure/core/implementation/annotation/ResponseType.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.annotation;
+
+/**
+ * Enumeration of return types used with {@link ServiceMethod} annotation to indicate if a
+ * method is expected to return a single item or a collection
+ */
+public enum ResponseType {
+    SINGLE_ITEM,
+    COLLECTION_OF_ITEMS
+}
```

Comment:
I'm not a fan of the enum name or the name of the values. I would instead have considered calling it `ReturnType` and maybe called the values `SINGLE` and `COLLECTION`.